docs_user/assemblies/assembly_adopting-the-data-plane.adoc

```diff
@@ -6,12 +6,13 @@
 
 Adopting the {rhos_long} data plane involves the following steps:
 
-. Stopping any remaining services on the {rhos_prev_long} ({OpenStackShort}) control plane.
-. Deploying the required custom resources.
-. If applicable, performing a fast-forward upgrade on Compute services from {OpenStackShort} {rhos_prev_ver} to {rhos_acro} {rhos_curr_ver}.
+. Stop any remaining services on the {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver} control plane.
+. Deploy the required custom resources.
+. If applicable, perform a fast-forward upgrade on Compute services from {OpenStackShort} {rhos_prev_ver} to {rhos_acro} {rhos_curr_ver}.
+. If applicable, adopt Networker nodes to the {rhos_acro} data plane.
```

Comment:
Hi @klgill I notice that there are lots of instances of the word Networker, this is new to me but I presume this is correct.

Reply:
it should be "network node"  like "compute node" so i would likely replace networker with network in all cases.

a "network node" is a  host dedicated exclusively for running neutron services like l3 agents, DHCP ectra. its less common now that we have ovn as our default backend but it was sometimes desirable to have separate network nodes when using ironic or for centralised snat in the past instead of putting that traffic through the controller nodes.

Reply:
I've seen Networker node in our documentation, e.g., it's used throughout the "Configuring Red Hat OpenStack Platform networking" guide: https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html-single/configuring_red_hat_openstack_platform_networking/index#networking_service_components_network-overview

That's recent enough that I'll want to confirm with the Networking group.